src/css/main.css

```diff
@@ -34,6 +34,9 @@ a:hover {
 
 .navbar {
   z-index: 1; /* Overlap the main content */
+  -webkit-box-shadow: 0px 1px 5px 0px rgba(0,0,0,0.75);
```

Comment:
You can remove these vendor prefixed properties -moz and -webkit; they will be added automatically if needed for the target browsers by the build (autoprefixer). See the constant `TARGET_BROWSERS` in `config/postcss.config.js` for the target browsers as they are now.